src/status_im/chat/models/images.cljs

```diff
@@ -51,18 +52,22 @@
  ::chat-open-image-picker
  (fn []
    (react/show-image-picker
-    (fn [result]
-      (resize-and-call
-       (aget result "path")
-       #(re-frame/dispatch [:chat.ui/image-selected %])))
-    "photo")))
+    (fn [images]
+      (when (pos? (count images))
+        (re-frame/dispatch [:chat.ui/clear-sending-images]))
+      (doseq [result (take max-images-batch images)]
+        (resize-and-call
+         (aget result "path")
```

Comment:
Why do we use `aget` instead of js introp?

Reply:
It was like that, I just wrapped it in a `doseq`. Now it doesn't matter, but I see your point. IMHO it will be nice to have a code style document or write custom lint rules to catch such things.

Reply:
Wasn't `(aget result "localIdentifier")` added in this PR? Doc would be nice but I also see no point in multiplying this `aget` thing here